### ISSUE TITLE
Move Delete comment button to the left

### ISSUE THREAD

**Issue by ioanarusiczki** [mozilla/addons-code-manager#1147]
Followup for https://github.com/mozilla/addons-code-manager/issues/714#issuecomment-544557057

Until https://github.com/mozilla/addons-code-manager/issues/997 is fixed the "Delete" button for a comment should be moved to the left because:
- when the code line is long the button is not visible on the page.
- resizing the browser the button won’t be visible unless I do not collapse the side panels.

Examples:

![scroll to the right to see delete](https://user-images.githubusercontent.com/33448286/67267694-80e40680-f4bb-11e9-8644-dda844e634c2.gif)

![resize the screen](https://user-images.githubusercontent.com/33448286/67267704-880b1480-f4bb-11e9-81db-2b56af38c5e3.gif)

This issue is reproducible on dev with FF69(Win10).

**Comment by ioanarusiczki** [comment]
@kumar303 Verified on dev with FF70 (Win10). 

![moved to the left](https://user-images.githubusercontent.com/33448286/67773175-faec3080-fa63-11e9-9c7f-0f6d8ec55e90.gif)
